<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainClass</class>
 <widget class="QWidget" name="MainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoYu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="main_wgt" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QWidget" name="left_wgt" native="true">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_novel">
           <property name="text">
            <string>Novel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_image">
           <property name="text">
            <string>Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_video">
           <property name="text">
            <string>Video</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_iptv">
           <property name="text">
            <string>IPTV</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_jiexi">
           <property name="text">
            <string>Jiexi</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_favorite">
           <property name="text">
            <string>Favorite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_setting">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="windows_wgt" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTabWidget" name="body_wgt">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_novel">
         <attribute name="title">
          <string>Novel</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="table_image">
         <attribute name="title">
          <string>Image</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_video">
         <attribute name="title">
          <string>Video</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>player</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_iptv">
         <attribute name="title">
          <string>IPTV</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>player</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_jiexi">
         <attribute name="title">
          <string>Jiexi</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>player</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_favorite">
         <attribute name="title">
          <string>Favorite</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_settings">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Manager</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
